pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook_test.go

```diff
@@ -953,14 +973,47 @@ func TestHandleNamespace(t *testing.T) {
 		resourceValidator fleetResourceValidator
 		wantResponse      admission.Response
 	}{
-		"allow user to modify non-reserved namespace": {
+		"allow any user to modify non-reserved namespace": {
 			req: admission.Request{
 				AdmissionRequest: admissionv1.AdmissionRequest{
-					Name:      "test-namespace",
-					Operation: admissionv1.Create,
+					Name: "test-namespace",
+					UserInfo: authenticationv1.UserInfo{
+						Username: "testUser",
+						Groups:   []string{"testGroup"},
+					},
+					RequestKind: &utils.NamespaceMetaGVK,
+					Operation:   admissionv1.Create,
+				},
+			},
+			wantResponse: admission.Allowed(allowedMessageNonReservedNamespace),
+		},
+		"allow any user to modify non-reserved namespace with fleet prefix": {
```

Comment:
Hi Arvind! Just another nit: the name's a bit confusing to me, would something like `allow any user to modify non-reserved namespace` or `allow any user to modify namespace without fleet- prefix` be more clear?